.github/workflows/release.yml

```diff
@@ -147,22 +155,13 @@ jobs:
       - name: Extract branch name
```

Comment:
Are we doing step this again?  Job at L#79 also has the same id.

Reply:
we are doing this step again for different jobs, this comes under platform executable job, and L#79 comes under release job. We can have a job which extracts branch-name and use its reference at different places (oclif-release job, docker release job). But since I see we are just extracting it from GITHUB_REF (not computation heavy), it doesn't seems necessory. Let me know your thoughts

Reply:
I see that this has been used in platform exec and docker release too. I am okay with having this.